### PR TITLE
Continuous actions bindings

### PR DESCRIPTION
**Pull request by olgavrou**
Mirroring the other bindings from the other loops

Next step:
c# simulator example (in `RLSimulator.cs`) for continuous actions

### CODE REVIEW

**Review by kumpera — CHANGES_REQUESTED** [review]

**Comment by kumpera** [comment]
Looks great, small issue around build files, everything else are just suggestions.

File: bindings/cs/rl.net.native/rl.net.native.vcxproj [hunk 1]
```diff
@@ -96,6 +96,7 @@
     <ClCompile Include="rl.net.api_status.cc" />
     <ClCompile Include="rl.net.buffer.cc" />
     <ClCompile Include="rl.net.config.cc" />
+    <ClCompile Include="rl.net.continuous_action_response.cc" />
```
[the remark this answers is not in the document]
Comment (olgavrou) [reply bindings/cs/rl.net.native/rl.net.native.vcxproj:99]: It's a shame the CI doesn't catch this

File: bindings/cs/rl.net/ContinuousActionResponse.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            get
+            {
+                float result = NativeMethods.GetContinuousChosenAction(this.DangerousGetHandle());
+
+                GC.KeepAlive(this);
+                return result;
+            }
+        }
+
+        public float ChosenActionPdfValue
```
[the remark this answers is not in the document]
Comment (olgavrou) [reply bindings/cs/rl.net/ContinuousActionResponse.cs]: Replaced and the Get method then becomes `GetContinuousActionChosenActionDensity`

File: bindings/cs/rl.net.native/rl.net.native.vcxproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (olgavrou) [reply bindings/cs/rl.net.native/rl.net.native.vcxproj:99]: Done

File: bindings/cs/rl.net.native/rl.net.continuous_action_response.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    API reinforcement_learning::continuous_action_response* CreateContinuousActionResponse();
+    API void DeleteContinuousActionResponse(reinforcement_learning::continuous_action_response* response);
+
+    // TODO: We should think about how to avoid extra string copies; ideally, err constants
+    // should be able to be shared between native/managed, but not clear if this is possible
+    // right now.
+    API const char* GetContinuousActionEventId(reinforcement_learning::continuous_action_response* response);
+    API const char* GetContinuousActionModelId(reinforcement_learning::continuous_action_response* response);
+
+    API float GetContinuousChosenAction(reinforcement_learning::continuous_action_response* response);
```
[the remark this answers is not in the document]
Comment (olgavrou) [reply bindings/cs/rl.net.native/rl.net.continuous_action_response.h]: Replaced

**Pull request merged**